while 1:
	x=1
	print('error')
